todos/src/app/home/list/list.component.ts

```diff
@@ -34,11 +34,11 @@ export class ListComponent {
   titleState = true
   
 
-  translate() {
+  async translate() {
     for (let index = 0; index < this.list.cards.length; index++) {
       let contentCard = this.list.cards[index].content;
       let cardId = this.list.cards[index].id;
-      this.serviceTranslate.translateApi(contentCard,cardId).then((data) => {
+      await this.serviceTranslate.translateApi(contentCard,cardId).then((data) => {
```

Comment:
You left the `.then()` there, you need to assign the result of the await to a variable and use that.